### ISSUE TITLE
NSFW commands do not work

### ISSUE THREAD

**Issue by phosphene47** [1337r34p3r/neko-life-bot#1]
Currently NSFW channel ID is hardcoded in `bot.js` and no NSFW command works outside that channel.
It should reply in the same channel as the command after checking if the channel is labelled NSFW, but Discord.io offers no way to do that afaik.

**Comment by HMH-CGZ-Shubhrit-Dhanawat** [comment]
Yeah there is no way to find for now using discord.io. Atleast I can't find it in the documentation.

**Comment by 1337r34p3r** [comment]
In discord.io documentation I couldn't find any way to know if the channel is NSFW or not. That's why I have hardcoded it to a specific nsfw channel. Later if I move to other library I will fix it.
